src/main/java/kernbeisser/DBEntities/PreOrder.java

```diff
@@ -96,6 +96,9 @@ public String toString() {
 
   @Override
   public boolean isInRelation(@NotNull User user) {
+    if (this.createDate == null) { // must be a new preorder that cannot have a user relation
```

Comment:
move the User as a required argument in the constructor of the PreOrder -> a PreOrder can only exits with a User who ordered it. Otherwise we can make use of Access.getExceptions() and add an exception to the PreOrder which should become added.

Reply:
That's feasible with pre-orders. On the other hand, there is no reason, why someone should not access any object, he has crested himself...
E.g. when it comes to adding users without having general access to users, we need something, like "I am related to a new object anyway